tests/python/gpu/test_operator_gpu.py

```diff
@@ -1975,6 +1975,19 @@ def test_bilinear_sampler_versions():
         assert_almost_equal(exe_list[ref_idx].grad_dict['data'].asnumpy(), data_grad + data_initial_grad, rtol=1e-3, atol=1e-5)
         assert_almost_equal(exe_list[ref_idx].grad_dict['grid'].asnumpy(), grid_grad + grid_initial_grad, rtol=1e-3, atol=1e-5)
 
+        req_dict = {'data' : 'null', 'grid' : 'write'}
```

Comment:
not testing req1 != Null ?

Reply:
Well the actual use case reported on the issues is this one, but maybe I should add the other one even if it's not a reported use case at all.

Reply:
Added extra test case as per your review, should be good for merge.